source/extensions/filters/http/ip_tagging/ip_tagging_filter.cc

```diff
@@ -51,11 +51,17 @@ IpTaggingFilterConfig::IpTaggingFilterConfig(
     tag_data.emplace_back(ip_tag.ip_tag_name(), cidr_set);
   }
   trie_ = std::make_unique<Network::LcTrie::LcTrie<std::string>>(tag_data);
+  // TODO(jmarantz): save stat-names for each remote address as stat_name_set builtins.
```

Comment:
You mean remote tag?

Reply:
Not sure; I used 'address' here as trie is constructed from:
```
  std::vector<std::string> tags =
      config_->trie().getData(callbacks_->streamInfo().downstreamRemoteAddress());
```
but happy to use whatever terminology you think makes the most sense. Changed.